docs/notebooks/nb_06_integrals.ipynb

```diff
@@ -0,0 +1,171 @@
+{
```

Comment:
<p>Some of the equations are not displaying on my computer. For example, </p><p> </p><p>\begin{equation} &nbsp;\langle pq | rs \rangle = \int dx_1 dx_2 \phi_p^<em>(x_1) \phi_q^</em>(x_2) r_{12}^{-1} \phi_r(x_1) \phi_s(x_2) &nbsp;\end{equation} </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/evangelistalab/forte/pull/246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>